README.md

```diff
@@ -32,6 +32,33 @@ integers, please see the [notes on atomicity](#atomicity).
 
 Rayon currently requires `rustc 1.12.0` or greater.
 
+### Using Rayon
+
+[Rayon is available on crates.io](https://crates.io/crates/rayon). The
+recommended way to use it is to add a line into your Cargo.toml such
+as:
+
+```rust
+[dependencies]
+rayon = XXX # <-- insert latest version of Rayon from crates.io here
```

Comment:
Here I think we should fill in the current `"0.8"` semver, so users don't have to figure that out.  It's a relatively small thing for us to update when we do bump semver.